miasm2/expression/expression.py

```diff
@@ -474,9 +496,9 @@ def get_modified_slice(self):
         modified_s = []
         for x in self.src.args:
             if (not isinstance(x[0], ExprSlice) or
-                x[0].arg != dst    or
+                x[0].arg != dst or
                 x[1] != x[0].start or
-                x[2] != x[0].stop):
+                    x[2] != x[0].stop):
```

Comment:
Actually, `autopep8` on:
```Python
if (not isinstance(x[0], ExprSlice) or
    x[0].arg != dst or
    x[1] != x[0].start or
        x[2] != x[0].stop):
```

Does not change anything (as you remark).
But, on this code:
```Python
if (not isinstance(x[0], ExprSlice) or
    x[0].arg != dst or
    x[1] != x[0].start or
    x[2] != x[0].stop):
```
It results in:
```Python
if (not isinstance(x[0], ExprSlice) or
        x[0].arg != dst or
        x[1] != x[0].start or
        x[2] != x[0].stop):
```

Which is, for me, easier to read and homogeneous.